README.md

```diff
@@ -100,12 +100,13 @@ Re-run `. .venv/bin/activate` in each shell.
     aws-azure-login --no-prompt --profile=<AWS-PROFILE-NAME>
     ```
 1. Environment variables
-* **DEPLOY_ENV:** set deployment environment. Recommended values: prod, nonprod, ci, dev or any
-    string without spaces. Default: dev.
-* **DATALAKE_USE_EXISTING_VPC:** determine if networking stack will use existing VPC in target AWS
-    account or will create a new one. Existing VPC must to be used must contain following tags -
-    "ApplicationName": "geospatial-data-lake", "ApplicationLayer": "networking".
-    Allowed values: true, false. Default: false.
+   * **`DEPLOY_ENV`:** set deployment environment. 
+     For your personal development stack: set DEPLOY_ENV to your username. 
+     
+     ```bash
+     export DEPLOY_ENV="$USER"
+     ```
+     Other values ued by CI pipelines include: prod, nonprod, ci, dev or any string without spaces. Default: dev.
```

Comment:
Hey @imincik , thanks for your comments.
1.  I agree that this should be documented.  My intent with the change was to remove different code paths for LINZ/non-LINZ developers that are controlled by environment variables. I think this adds unnecessary complexity. I've added the networking_stack.py back in to the project to document the networking requirements and added steps into the readme to help potential other users of the application.  [commit of changes](https://github.com/linz/geospatial-data-lake/pull/198/commits/20d519c9045ca491e97d9d2b36ca9c0bd52b26f8)

Reply:
Are we ever running this networking stack code? As in, all of it? If not I would strongly recommend removing it, since the odds that it is actually working and completely in sync with our networking stack is practically nil.